.gitignore

```diff
@@ -67,3 +67,7 @@ tsconfig.tsbuildinfo
 
 # Config Cache Directory
 config/cache
+
+# Yarn
+yarn.lock
+.yarn/*
```

Comment:
having `yarn.lock` in the gitignore is not recommended because it's used for consistent installs. more can be read about this here: https://classic.yarnpkg.com/lang/en/docs/yarn-lock/

the `.yarn` folder probably doesn't matter as much for yarn v1 but it might not be ideal. while overseerr does in fact use yarn v1, newer versions of yarn (v2+) requires this folder to be version controlled.

Reply:
Ok, I did add them back 👍